gno.land/cmd/gnoland/config_set_test.go

```diff
@@ -0,0 +1,900 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gnolang/gno/tm2/pkg/bft/config"
+	"github.com/gnolang/gno/tm2/pkg/bft/state/eventstore/file"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"github.com/gnolang/gno/tm2/pkg/db"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// initializeTestConfig initializes a default configuration
+// at a temporary path
+func initializeTestConfig(t *testing.T) string {
+	t.Helper()
+
+	path := filepath.Join(t.TempDir(), "config.toml")
+	cfg := config.DefaultConfig()
+
+	require.NoError(t, config.WriteConfigFile(path, cfg))
+
```

Comment:
Btw suggest adding a cleanup function to remove the tempdir ([testing.T.Cleanup](https://pkg.go.dev/testing#T.Cleanup))

Reply:
I utilize [t.TempDir](https://pkg.go.dev/testing#T.TempDir) for the top-level config directory, which is auto wiped on every test completion